### PR TITLE
Bump pulsar to 2.7.0

### PR DESCRIPTION
**Pull request by BewareMyPower**
Bump pulsar to 2.7.0. Since BookKeeper API has changed, the KoP will be incompatible with Pulsar 2.6.x after this PR.

### CODE REVIEW

**Comment by BewareMyPower** [comment]
After this PR is merged, we'll release KoP 2.7.0.

Then we need to bump to the latest pulsar (e.g. `2.8.0-rc-202012200040`) because we need some latest features of pulsar, like what #263 needed.

**Comment by BewareMyPower** [comment]
Wait for #279 merged

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**
